ynr/apps/popolo/admin.py

```diff
@@ -4,11 +4,6 @@
 
 from .behaviors import admin as generics
 
-try:
-    pass
-except ImportError:
-    pass
-
```

Comment:
lol. Any idea what this was ever supposed to achieve? :laughing:

Reply:
First time I'd seen this too – no idea!